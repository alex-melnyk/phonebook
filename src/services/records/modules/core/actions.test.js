import {
  createSuccess,
  editSuccess,
  remove,
} from './actions';

describe('modal actions', () => {

  it('should create correctly', () => {

    expect(createSuccess()).toMatchSnapshot();

  });

  it('should edit correctly', () => {

    expect(editSuccess()).toMatchSnapshot();

  });

  it('should remove correctly', () => {

    expect(remove()).toMatchSnapshot();

  });

});
