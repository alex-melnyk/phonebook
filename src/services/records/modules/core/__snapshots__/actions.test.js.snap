// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal actions should create correctly 1`] = `
Object {
  "error": false,
  "payload": undefined,
  "type": "RECORDS_CORE_CREATE_SUCCESS",
}
`;

exports[`modal actions should edit correctly 1`] = `
Object {
  "error": false,
  "payload": undefined,
  "type": "RECORDS_CORE_EDIT_SUCCESS",
}
`;

exports[`modal actions should remove correctly 1`] = `
Object {
  "error": false,
  "payload": undefined,
  "type": "RECORDS_CORE_REMOVE",
}
`;
